src/mame/drivers/vicdual.cpp

```diff
@@ -3909,7 +3909,7 @@ GAME( 1981, brdrlins,   brdrline, brdrline,  brdrline,  vicdual_state, empty_ini
 GAME( 1981, brdrlinb,   brdrline, brdrline,  brdrline,  vicdual_state, empty_init, ROT270, "bootleg (Karateco)", "Borderline (Karateco bootleg)", MACHINE_NO_SOUND | MACHINE_SUPPORTS_SAVE )
 GAME( 1981, brdrlinet,  brdrline, tranqgun,  tranqgun,  vicdual_state, empty_init, ROT270, "Sega", "Borderline (Tranquillizer Gun conversion)", MACHINE_NOT_WORKING | MACHINE_IMPERFECT_SOUND | MACHINE_SUPPORTS_SAVE ) // official factory conversion
 GAME( 198?, startrks,   0,        headons,   headons,   vicdual_state, empty_init, ROT0,   "bootleg (Sidam)", "Star Trek (Head On hardware)", MACHINE_IMPERFECT_SOUND | MACHINE_SUPPORTS_SAVE )
-GAME( 1980, digger,     0,        digger,    digger,    vicdual_state, empty_init, ROT270, "Sega", "Digger", MACHINE_NO_SOUND | MACHINE_SUPPORTS_SAVE )
+GAME( 1980, digger,     0,        digger,    digger,    vicdual_state, empty_init, ROT270, "Gremlin / Sega", "Digger", MACHINE_NO_SOUND | MACHINE_SUPPORTS_SAVE )
```

Comment:
If this was a work-for-hire and only sold by Sega with Sega branding, we wouldn't list the developer here.  Is there any evidence this was actually marketed as a Gremlin product?

Reply:
Is there any evidence this was actually marketed as a Gremlin product?
Gremlin / Sega is shown on the cabinet:
https://www.arcade-museum.com/game_detail.php?game_id=7583